packages/employer-api/src/utils/addressToCatchment.ts

```diff
@@ -0,0 +1,27 @@
+/* eslint-disable import/prefer-default-export */
+
+import pins from "../constants/centres.json"
```

Comment:
We need to remove catchments 46 and 47 from the list of options in centres.json.